<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
    border-image: url(&quot;:/1/tv.png&quot;) 0 0 0 0 stretch stretch;
	background-color: transparent;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>521</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>80</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="seleccionetexto">
         <property name="styleSheet">
          <string notr="true">QLabel {

	font: 8pt &quot;Times New Roman&quot;;
    font-size: 20pt;
    color: black;
}
</string>
         </property>
         <property name="text">
          <string>Seleccione la llave con el arreglo a manejar:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: white;
    color: black;
    border: 1px solid black;
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 12pt;
}

QComboBox:hover {
    border: 1px solid #3498db;
}


</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="continuarLlave">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 1px solid black;
    border-radius: 6px;
    padding: 8px 16px;
    font-size: 12pt;
}

QPushButton:hover {
    background-color: #ecf0f1; 
    border-color: #34495e;
    color: #34495e;
}

QPushButton:pressed {
    background-color: #bdc3c7; 
    border-color: #2c3e50;
    color: #2c3e50;
}</string>
         </property>
         <property name="text">
          <string>Continuar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="volverBoton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 1px solid black;
    border-radius: 3px;
    padding: 7px 10px;
    font-size: 10pt;
}

QPushButton:hover {
    background-color: #ecf0f1; 
    border-color: #34495e;
    color: #34495e;
}

QPushButton:pressed {
    background-color: #bdc3c7; 
    border-color: #2c3e50;
    color: #2c3e50;
}</string>
    </property>
    <property name="text">
     <string>VOLVER</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Imagenes/bgPrueba.qrc"/>
  <include location="../fuentes/fuentes.qrc"/>
 </resources>
 <connections/>
</ui>
